velox/functions/sparksql/RegexFunctions.h

```diff
@@ -60,4 +60,28 @@ std::shared_ptr<exec::VectorFunction> makeRegexExtract(
     const std::vector<exec::VectorFunctionArg>& inputArgs,
     const core::QueryConfig& config);
 
+std::shared_ptr<exec::VectorFunction> makeRegexReplace(
```

Comment:
makeRegexReplace and makeRe2Replace function names appear the "same". What are the differences between these 2 functions? Would it be possible to come up with more descriptive names? Also, please, document makeRegexReplace function.

Reply:
makeRe2Replace is a forgotten function and not defined under RegexFunctions.cpp. Will remove with next commit. Only replace function is makeRegexReplace. Will document with next commit as well.

Reply:
Removed in recent commit